packages/ragel.rb

```diff
@@ -0,0 +1,18 @@
+require 'package'
+
+class Ragel < Package
+  description 'ragel desc stub'
```

Comment:
Can we change this description?  Does `Ragel State Machine Compiler` make it more clear?

Reply:
Dang it, forgot to change that. Fixing now.